app/service/rest.py

```diff
@@ -945,10 +948,35 @@ def check_if_reply_to_address_already_in_use(service_id, email_address):
 
 @service_blueprint.route('/<uuid:service_id>/returned-letter-summary', methods=['GET'])
 def returned_letter_summary(service_id):
-    results = get_returned_letter_summary(service_id)
+    results = fetch_returned_letter_summary(service_id)
 
     json_results = [{'returned_letter_count': x.returned_letter_count,
                      'reported_at': x.reported_at.strftime(DATE_FORMAT)
                      } for x in results]
 
     return jsonify(json_results)
+
+
+@service_blueprint.route('/<uuid:service_id>/returned-letters', methods=['GET'])
+def get_returned_letters(service_id):
+    results = fetch_returned_letters(service_id=service_id, report_date=request.args.get('reported_at'))
+
+    json_results = [
+        {'notification_id': x.notification_id,
+         # client reference can only be added on API letters
+         'client_reference': x.client_reference if x.api_key_id else None,
+         'reported_at': x.reported_at.strftime(DATE_FORMAT),
+         'created_at': x.created_at.strftime("%Y-%m-%d %H:%M:%S"),
```

Comment:
Should we be using `DATETIME_FORMAT` here?

Reply:
No - but I should add a new constant for REPORT_DATETIME_FORMAT

Reply:
Is there a quick explanation for why `DATETIME_FORMAT` isn't appropriate?  Like we can't send that and then format it on the frontend if we need to do it differently?

Reply:
Seems that the dates sent to the frontend are in this format.